### PR TITLE
[4.3] Fix cookie with Max-Age processing

### PR DESCRIPTION
**Pull request by stlaz**
When cookie has Max-Age set it tries to get expiration by adding
to a timestamp. Without this patch the timestamp would be set to
None and thus the addition of timestamp + max_age fails

https://pagure.io/freeipa/issue/6718

### CODE REVIEW

File: ipalib/rpc.py [hunk 1]
```diff
@@ -788,8 +791,10 @@ def get_session_cookie_from_persistent_storage(self, principal):
 
         # Search for the session cookie within the cookie string
         try:
-            session_cookie = Cookie.get_named_cookie_from_string(cookie_string, COOKIE_NAME)
-        except Exception as e:
+            session_cookie = Cookie.get_named_cookie_from_string(
+                cookie_string, COOKIE_NAME,
+                timestamp=datetime.datetime.utcnow())
+        except Exception:
```
Comment (MartinBasti) [inline ipalib/rpc.py]: Debug log here might be useful, right?

Comment (stlaz) [inline ipalib/rpc.py]: Sure, why not. I was just porting the code not really looking that way or another.

Comment (MartinBasti) [inline ipalib/rpc.py]: In that case, create another commit that can be applied to master and 4.5 branch as well

**Comment by MartinBasti** [comment]
Please open a new ticket `Backport ...`. Ticket you used is closed in closed milestone

**Comment by stlaz** [comment]
The ticket's already there, I just recycled the old commit message and forgot to replace it.

**Comment by tomaskrizek** [comment]
ipa-4-3:

* 0d66046e501a4a1a09a0a74a96a499cb88ffb03b Fix cookie with Max-Age processing
* 71475e3153117e554d22a2a27d7882ba4f890be8 Add debug log in case cookie retrieval went wrong

**Pull request closed without merging**